pmultiqc_service/app.py

```
@@ -729,5 +950,220 @@
+                'finished_at_seconds': current_time()[0],
+                'finished_at': current_time()[1]
+            }
+            return
+        
+        job_status_dict[job_id] = {'status': 'extracting_files', 'progress': 70}
+        
+        # Process each downloaded file separately
+        all_results = []
+        total_processed = 0
+        
+        for i, zip_file in enumerate(downloaded_files):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

(B007)